oxen-python/README.md

```
@@ -41,7 +41,7 @@ repo.checkout()
 If you don't want to download the data locally, you can use the `RemoteRepo` class to interact with a remote repository on OxenHub.
 
 ```python
-import oxen 
+import oxen
 
 repo = RemoteRepo("https://hub.oxen.ai/ox/CatDogBBox")
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)